### PR TITLE
[CodeCamp2023-475] Add new configuration files for Mask2Former algorithm in mmdetection.

### PR DESCRIPTION
**Pull request by ChengAoShen**
## Motivation

To finish the task in CodeCamp2023

## Modification

Add new configuration files for Mask2Former

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMPreTrain.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: mmdet/configs/mask2former/mask2former_r101_8xb2_lsj_50e_coco.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from mmengine.config import read_base
+
+with read_base():
+    from .mask2former_r50_8xb2_lsj_50e_coco import *
+
+model.merge(
```
Comment (hhaAndroid) [inline mmdet/configs/mask2former/mask2former_r101_8xb2_lsj_50e_coco.py:7]:
```suggestion
model.update(
```

File: mmdet/configs/mask2former/mask2former_r50_8xb2_lsj_50e_coco.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        type=DetDataPreprocessor,
+        mean=[123.675, 116.28, 103.53],
+        std=[58.395, 57.12, 57.375],
+        bgr_to_rgb=True,
+        pad_size_divisor=32,
+        pad_mask=True,
+        mask_pad_value=0,
+        pad_seg=False,
+        batch_augments=batch_augments))
+model.merge(
```
Comment (hhaAndroid) [inline mmdet/configs/mask2former/mask2former_r50_8xb2_lsj_50e_coco.py:43]: 所有的 merge 全部改成 update，这样更符合 python 用法

**Comment by hhaAndroid** [comment]
@ChengAoShen Please fix the CI

**Comment by OpenMMLab-Assistant-004** [comment]
Hi @ChengAoShen,

We'd like to express our appreciation for your valuable contributions to the mmdetection. Your efforts have significantly aided in enhancing the project's quality.
It is our pleasure to invite you to join our community thorugh Discord_Special Interest Group (SIG) channel. This is a great place to share your experiences, discuss ideas, and connect with other like-minded people. To become a part of the SIG channel, send a message to the moderator, OpenMMLab, briefly introduce yourself and mention your open-source contributions in the #introductions channel. Our team will gladly facilitate your entry. We eagerly await your presence. Please follow this link to join us: ​https://discord.gg/UjgXkPWNqA.

If you're on WeChat, we'd also love for you to join our community there. Just add our assistant using the WeChat ID: openmmlabwx. When sending the friend request, remember to include the remark "mmsig + Github ID".

Thanks again for your awesome contribution, and we're excited to have you as part of our community!

**Comment by hhaAndroid** [comment]
@ChengAoShen Hi. Do you have any progress?

**Pull request closed without merging**